plugins/nextcloud-rule-exclusions-before.conf

```
@@ -1062,6 +1062,22 @@ SecRule REQUEST_FILENAME "@rx /apps/photos/api/v[0-9]/config/config/(?:croppedLa
     ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
     setvar:'tx.allowed_methods=%{tx.allowed_methods} PUT'"
 
+# Photos: Albums, Shared Albums, Places
+# Allow the data type 'text/plain'
+# Since the content is actually XML, we switch on the XML parser
+SecRule REQUEST_FILENAME "@rx /remote\.php/dav/photos/[^/]+/(?:albums|sharedalbums|places)(?:/[^/]+)?/$" \
+    "id:9508951,\
```

I just realized that rule id ``9508951`` is already in use, you'll have to pick another rule id like ``9508952`` to avoid a conflict.
https://github.com/coreruleset/nextcloud-rule-exclusions-plugin/blob/327425915e1ca48accaaba6b8ca06fd4f13cfc82/plugins/nextcloud-rule-exclusions-before.conf#L1216
```suggestion
    "id:9508952,\
```

---

Just fixed :)